src/components/Footer/styles.module.scss

```diff
@@ -0,0 +1,230 @@
+@import '../../app/variables';
+@import '../../app/mixins';
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@800&display=swap');
```

Comment:
Nice butt. there are a few errors with this approach, ok we know the following, the roboto and Montserrat fonts should be used in the whole system. you need to set up this font globally in you use this approach you only load the font for the footer. and another issue about the use of this approach has to do with performance.
ok the correct way to do this is using the next fonts.
I recommended this article https://nextjs.org/docs/app/building-your-application/optimizing/fonts  please review and adjust your code following the correct approach

Reply:
Good evening engineer.
I modified what you asked for.
Please check my PR?